index.html

```diff
@@ -0,0 +1,484 @@
+
+        <div class="btn-more">
+          <button class="more" type="button">
+            MORE <i class="bi bi-chevron-down down-arrow"></i>
+          </button>
+        </div>
+
+        <div class="btn-less">
+          <button class="less" type="button">
+            LESS <i class="bi bi-chevron-up down-arrow"></i>
+          </button>
+        </div>
+      </section>
+
+      <section id="partner" class="partner">
+        <h2 class="partner-title">Partner</h2>
+        <p class="partner-line"></p>
+
+        <ul class="partners-list">
+          <li>
+            <a
+              class="column"
+              href="https://www.mozilla.org/ru/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                alt="mozilla"
+                src="images/partner/mozilla.png"
+                height="50"
+                width="200"
+              />
+            </a>
+          </li>
+          <li>
+            <a
+              class="column"
+              href="https://www.google.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                alt="Google"
+                src="images/partner/Google.png"
+                height="60"
+                width="200"
+              />
+            </a>
+          </li>
+          <li>
+            <a
+              class="column"
+              href="https://www.naver.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                alt="naver"
+                src="images/partner/navar.png"
+                height="35"
+                width="200"
+              />
+            </a>
+          </li>
+          <li>
+            <a
+              class="column"
+              href="https://www.behance.net/gallery/70720545/DaumKakao-Brand-Products-for-Biz-Division"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                alt="daumkakao"
+                src="images/partner/daumkakao.svg"
+                height="50"
+                width="200"
+              />
+            </a>
+          </li>
+          <li>
+            <a
+              class="column"
+              href="https://ru.airbnb.com/?_set_bev_on_new_domain=1641255426_NzBkOWFmNzUzMDFi"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                alt="airbnb"
+                src="images/partner/airbnb.png"
+                height="50"
+                width="200"
+              />
+            </a>
+          </li>
+        </ul>
+      </section>
+
+      <div class="end">
+        <a class="logo" href="#home">
+          <img alt="logo" src="images/logo.png" height="45" width="90" />
+        </a>
+        <h2>2022 Creative Student &copy; Some Rights Reserved</h2>
+      </div>
```

Comment:
- [ ] [OPTIONAL] Hi, on the footer section please notice that the logo and text are intersecting with each other. Please kindly fix this each.
![image](https://user-images.githubusercontent.com/18730106/148290968-47edb404-6ab5-42cd-af66-2b0374d81216.png)